### PR TITLE
Move `Key` from `react-table` to `react`

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky @sgrishchenko

### CODE REVIEW

File: kotlin-react/src/main/kotlin/react/Props.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ external interface RProps
 val RProps.children: Any
     get() = asDynamic().children
 
-var RProps.key: String
+var RProps.key: ReactKey
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/Props.kt:8]: One more place?

File: kotlin-react/src/main/kotlin/react/Key.kt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package react
+
+typealias ReactKey = String
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/Key.kt:3]: `Key` like in types?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react/src/main/kotlin/react/Key.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-react/src/main/kotlin/react/Key.kt]: Done

File: kotlin-react/src/main/kotlin/react/Props.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-react/src/main/kotlin/react/Props.kt]: OK

**Review by turansky — APPROVED** [review]

**Comment by Leonya** [comment]
Released in pre.223

**Pull request merged**